velox/serializers/RowSerializer.h

```diff
@@ -23,10 +23,52 @@ namespace facebook::velox::serializer {
 
 using TRowSize = uint32_t;
 
+namespace detail {
+struct RowHeader {
+  int32_t uncompressedSize;
```

Comment:
Do we want the default values for these fields? Thanks!

Reply:
No, we don't know the value unless it is initialized.

Reply:
Then we should provide ctor?

Reply:
But we also need to set the field in RowHeader::read() one by one.